index.html

```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
 </div>
   </section>
 
-  <div class="section-cards">
+  <div>
+    <div class="project section-cards">
```

Comment:
On mobile:

- [ ] Please kindly add some margin or padding to the top of the popup content, so it looks like the Figma.
- [ ] Please align the buttons so they match the previous text alignment (see picture).
- [ ] Please make your popup a little bigger in height, so it matches the Figma.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/156447454-33773790-6b46-4bef-a084-8acfa232968f.png) | ![image](https://user-images.githubusercontent.com/57421823/156447495-a8e17dc4-cf58-4cc1-a991-7b5a719f048b.png)


If you implement these changes, your project will look more like the Figma 👍